package-lock.json

```diff
@@ -1050,17 +1050,17 @@
       "integrity": "sha512-x5HFsW+E/nQalGMw7hu+fvPqnBeBaIr0lWJ2SG0PPL2j+Pm9lYvCrsZJGIgauPIENx0v10INIyFjmSNUD/gSqQ=="
     },
     "@babel/plugin-external-helpers": {
-      "version": "7.8.3",
```

Comment:
I forgot to mention it earlier, please drop these changes. We usually update dependencies as dedicated PRs, unless a new fix / feature depends on the update.

Reply:
This has been reverted.